index.js

```diff
@@ -289,6 +289,15 @@ function SpotifyWebHelper(opts) {
 
       spotifyJsonRequest(this, '/remote/play.json', params, cb);
     }
+
+    this.getVersion = function(cb) {
+        var url = generateSpotifyUrl('/service/version.json');
+        return getJson(url, { 'service': 'remote' }, ORIGIN_HEADER, cb)
+    }
+
+    this.getLocalHostname = function() {
+      return generateRandomLocalHostName();
```

Comment:
What is the scenario in which you need this to be exposed to the calling module? It's was hidden since it's an implementation depdendant field..